docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Go (pull mode)"
+menuTitle: "Go (pull mode)"
+description:  "Instrumenting Golang applications for continuous profiling"
+weight: 10
+aliases:
+  - /docs/phlare/latest/configure-client/grafana-agent/
+---
+
+# Go (pull mode)
+
+The Grafana Agent is responsible for pulling profiles from applications and delivering them
+to the Pyroscope database.
+
+### Prerequisites
+
+To be able to pull profiles from applications, your applications needs to expose [pprof endpoints](https://pkg.go.dev/net/http/pprof).
+
+Before proceeding with the configuration, ensure that you have the following:
+
+1. Install Grafana Agent in [flow mode](/docs/agent/next/flow/setup/install/)
+2. Configure Grafana Agent in Flow mode
+
+## Adding Profiling to the Grafana Agent
+
+This guide presents how to configure the Grafana Agent for scraping performance profiles from a pprof endpoint using `pyroscope.scrape`. 
+
+**While this documentation will contain a simple example for more configuration options visit the [agent configuration documentation](/docs/agent/next/flow/reference/components/pyroscope.scrape/).**
```

Comment:
we should add a link to `/docs/agent/next/flow/reference/components/pyroscope.write/` somewhere as well

Reply:
or if we copy the whole page from the reference doc, than there's no need to link to the agent doc at all